app/jiraapi.py

```diff
@@ -159,25 +159,59 @@ def create_domain_approval_task(self, assessment):
         attachment.write(json.dumps(relevant_case_studies))
         self.generic_jira.jira.add_attachment(new_issue.id, attachment, 'casestudies.json')
 
-    def create_application_approval_task(self, application, closing_date=None):
+    def create_application_approval_task(self, application, domains, closing_date=None):
         if application.type != 'edit':
             summary = 'Application assessment: {}'.format(application.data.get('name'))
             description = TICKET_DESCRIPTION % (current_app.config['ADMIN_ADDRESS'] +
                                                 "/admin/applications/preview/{}".format(application.id))
             issuetype_name = INITIAL_ASSESSMENT_ISSUE_TYPE
         else:
             summary = 'Profile edit: {} (#{})'.format(application.data.get('name'), application.supplier_code)
-            description = "+*This is a profile edit*+\n\n" \
-                          "Please evaluate the changes made by the seller and ensure they meet the " \
-                          "[assessment guidelines|" \
-                          "https://govausites.atlassian.net/wiki/display/DM/Initial+Assessment+Checklist]. \n" \
-                          "Changes will be summarised at the top of the seller profile.\n\n" \
-                          "Seller profile link: [%s]\n\n" \
-                          "---\n\n" \
-                          "A snapshot of the application is attached." \
-                          "" % (current_app.config['ADMIN_ADDRESS'] +
-                                "/admin/applications/preview/{}".format(application.id))
+            description = (
+                "+*This is a profile edit*+\n\n"
+                "Please evaluate the changes made by the seller and ensure they meet the "
+                "[assessment guidelines|"
+                "https://govausites.atlassian.net/wiki/display/DM/Initial+Assessment+Checklist]. \n"
+                "Changes will be summarised at the top of the seller profile.\n\n"
+                "Seller profile link: [%s]\n\n"
+                "---\n\n"
+                "A snapshot of the application is attached."
+                "" % (current_app.config['ADMIN_ADDRESS'] + "/admin/applications/preview/{}".format(application.id))
+            )
             issuetype_name = SUBSEQUENT_ASSESSMENT_ISSUE_TYPE
+
+        pricing = application.data.get('pricing', None)
+        domain_price_min = 1
```

Comment:
not sure I follow why these need to be initialized

Reply:
removed